portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lorraine Lyu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="slide.css">
+    <script src="slide.js" defer></script>
+  </head>
+  <body>
+    <div id="content">
+      <div class="main">
+        <h1>My Image Gallary</h1>
+        <div class="slides" id="slide1">
+          <img class="pic pic-hide" src="images/cat0.jpg" alt="My Cat At His 4th Month">
+          <img class="pic pic-hide" src="images/cat1.jpg" alt="My Cat And My MoM">
+          <img class="pic pic-hide" src="images/cat2.jpg" alt="My Cat And His New Bed">
+          <img class="pic pic-hide" src="images/cat3.jpg" alt="My Cat Taking Over My Parents' Bed">
+          <img class="pic pic-hide" src="images/cat4.jpg" alt="My Cat On His First Birthday">
```

Comment:
Instead of having a 'pic-hide' class on all the photos, it would be simpler to apply a class to the 'active-picture.' Photos could be display: none by default and the new active class would overwrite that style to make it visible. When you change pictures, you can remove the active class from all img.pic elements and apply it to the correct one.